packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.internal.js

```diff
@@ -494,7 +494,7 @@ describe('ReactSuspenseWithNoopRenderer', () => {
         return <Text text="(empty)" />;
       }
       return (
-        <Suspense>
+        <Suspense fallback="Loading...">
```

Comment:
This test was testing the wrong thing, because a Suspense without a `fallback` will result in an error. So what this was actually testing was the "on error, try again at the lowest pending level."